content/docs/components-and-props.md

```diff
@@ -78,28 +78,28 @@ ReactDOM.render(
 
 [](codepen://components-and-props/rendering-a-component)
 
-Let's recap what happens in this example:
+Podsumujmy, co dzieje się w tym przykładzie:
 
-1. We call `ReactDOM.render()` with the `<Welcome name="Sara" />` element.
-2. React calls the `Welcome` component with `{name: 'Sara'}` as the props.
-3. Our `Welcome` component returns a `<h1>Hello, Sara</h1>` element as the result.
-4. React DOM efficiently updates the DOM to match `<h1>Hello, Sara</h1>`.
+1. Wywołujemy `ReactDOM.render()` z elementem `<Welcome name="Sara" />`.
+2. React wywołuje komponent `Welcome` z właściwościami `{name: 'Sara'}`.
+3. Nasz komponent `Welcome` jako wynik zwraca element `<h1>Cześć, Sara</h1>`.
+4. React DOM w optymalny sposób aktualizuje drzewo DOM, aby odpowiadało elementowi `<h1>Cześć, Sara</h1>`.
 
->**Note:** Always start component names with a capital letter.
+>**Wskazówka:** Zawsze zaczynaj nazwy komponentów od dużej litery.
```

Comment:
To ewentualnie zmieniłbym na `Uwaga`, tak jak np. tu: https://github.com/reactjs/pl.reactjs.org/pull/40/files#diff-6024d880e5eef39cfbff54653ef34fb6R61
`Wskazówka` nie jest właściwie niepoprawna, ale bardziej pasuje do angielskiego `Tip`.

```suggestion
>**Uwaga:** Zawsze zaczynaj nazwy komponentów od dużej litery.
```